server/users.ts

```diff
@@ -418,7 +419,7 @@ const SETTINGS = [
 
 // User
 export class User extends Chat.MessageContext {
```

Comment:
Wait, since when did this extend MessageContext? I'm going to need to investigate this.

Reply:
`git blame` reveals that it was added here: https://github.com/Zarel/Pokemon-Showdown/commit/eb67a12ad6c0f4021667468feedb3099ea255f1c#diff-68bb721f039d068f9977098b9f6c2ac2R437. I can't really tell _why_ it was needed but maybe @bumbadadabum can tell us.

Reply:
Oh, for translations, that makes sense.